rules/0520-vulnerability-detector_rules.xml

```diff
@@ -60,7 +60,7 @@
       <if_sid>23501</if_sid>
       <options>no_full_log</options>
       <field name="vulnerability.severity">Critical</field>
-      <description>$(vulnerability.title)</description>
+      <description>$(vulnerability.cve) affects $(vulnerability.package.name).</description>
```

Comment:
Remove the dot at the end of the description. Please do the same for all the rules of the file.

Reply:
DONE